WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -80,6 +91,44 @@ public class BlockEditorScreen: ScreenObject {
         return self
     }
 
+    public func addVideo() -> Self {
+        addMediaBlockFromUrl(
+            blockType: "Video block",
+            UrlPath: videoUrlPath
+        )
+
+        return self
+    }
+
+    public func addAudio() -> Self {
+        addMediaBlockFromUrl(
+            blockType: "Audio block",
+            UrlPath: audioUrlPath
+        )
+
+        return self
+    }
+
+    private func addMediaBlockFromUrl(blockType: String, UrlPath: String) {
+        addBlock(blockType)
+        insertFromUrlButton.tap()
+        app.textFields.element.typeText(UrlPath)
+        applyButton.tap()
+    }
+
+    @discardableResult
+    public func verifyMediaBlocksDisplayed() -> Self {
+        let imagePredicate = NSPredicate(format: "label CONTAINS[c] 'Image block'")
+        let videoPredicate = NSPredicate(format: "label CONTAINS[c] 'Video block'")
+        let audioPredicate = NSPredicate(format: "label CONTAINS[c] 'Audio block'")
```

Comment:
This function is called for a known state. IMO, there is no harm in making it more precise, which will also mean checking the blocks ordering:

```suggestion
        let imagePredicate = NSPredicate(format: "label == 'Image Block. Row 1'")
        let videoPredicate = NSPredicate(format: "label == 'Video Block. Row 2'")
        let audioPredicate = NSPredicate(format: "label == 'Audio Block. Row 3'")
```

(this is a working code)

Reply:
yup we can update this to make it more precise, updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20940/commits/8224bba762a536bcd1287ebaec1691b33f9658a9#diff-261df69291120d16908df389813290ce26e2af15c7c5f71c86d8c81b82d8024dR118-R120